apps/dotcms-ui/src/app/portlets/dot-edit-page/components/dot-palette/dot-palette-contentlets/dot-palette-contentlets.component.scss

```diff
@@ -0,0 +1,65 @@
+@use "libs/dot-primeng-theme-styles/src/scss/variables" as *;
```

Comment:
@hmoreras @rjvelazco I think we can do this in the block editor instead of the import I was doing.